certified-connectors/Flotiq/apiDefinition.swagger.json

```diff
@@ -0,0 +1,216 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Flotiq headless CMS",
+    "description": "Flotiq headless content management system",
```

Comment:
@andrzejwp Can you please provide the description value more in detailed. It should describe concisely the main purpose and value offered by your connector.

Documentation - [Description](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)

Reply:
Fixed.